db_scripts/init_db.sh

```diff
@@ -6,8 +6,8 @@ if [ $# -ge 1 ] ; then
   export KUBECONFIG=$1
 fi
 
-NS=esignet
-CHART_VERSION=12.0.2
+NS=inji-certify
+CHART_VERSION=0.0.1-develop
```

Comment:
should this not be 0.9.0-develop ? I am also not sure, lets check on this to devops team

Reply:
changed to 12.0.1 as suggested by devops team